### PR TITLE
[WIP] Wire up variables view object inspection

### PR DESCRIPTION
**Pull request by linclark**
This depends on #28.

### CODE REVIEW

File: devtools/client/webconsole/jsterm.js [hunk 1]
```diff
@@ -356,31 +356,37 @@ JSTerm.prototype = {
...
-    if (callback) {
-      let oldFlushCallback = this.hud._flushCallback;
-      this.hud._flushCallback = () => {
-        callback(msg.element);
-        if (oldFlushCallback) {
-          oldFlushCallback();
-          this.hud._flushCallback = oldFlushCallback;
-          return true;
-        }
+    if (this.hud.SUPER_FRONTEND_EXPERIMENT) {
```
Comment (linclark) [inline devtools/client/webconsole/jsterm.js]: early return

**Comment by linclark** [comment]
@bgrins we have variables view!!

I removed a little bit of test coverage. I'm going to land it as a separate test for prepareMessage, but I need to get the repeats PR in first (which depends on this one).

**Comment by bgrins** [comment]
Nice!! Let me know if I'm blocking you from getting anything landed

File: devtools/client/webconsole/new-console-output/components/message-container.js [hunk 2]
```diff
@@ -40,6 +44,18 @@ function getMessageComponent(messageType) {
   return MessageComponent;
 }
 
-module.exports.MessageContainer = MessageContainer;
+function mergeProps(stateProps, dispatchProps, ownProps) {
+  const jsterm = ownProps.jsterm;
+  const actor = ownProps.message.actor;
```
Comment (bgrins) [inline devtools/client/webconsole/new-console-output/components/message-container.js:49]: Does a message always have one actor?  What about with multiple `arguments`?

**Comment by bgrins** [comment]
Looks good to me overall, we could handle vview for multiple args in a follow up if changes are needed for that

**Comment by linclark** [comment]
That sounds good to me.

**Pull request merged**